docs/api/websocket_subscribe_api.md

```diff
@@ -41,13 +62,13 @@ Each API request has the following structure:
 }
 ```
 
-## Date Format
+## <a name='date-format'></a>Date Format
 
 In the contrary to the original Lisk Core API, all timestamps used by the Lisk Service are in the UNIX timestamp format. The blockchain dates are always expressed as integers and the epoch date is equal to the number of seconds since 1970-01-01 00:00:00.
```

Comment:
```suggestion
On the contrary to the original Lisk Core API, all timestamps used by the Lisk Service are in the UNIX timestamp format. The blockchain dates are always expressed as integers and the epoch date is equal to the number of seconds since 1970-01-01 00:00:00.
```